app/assets/stylesheets/wave-spinner.css.scss

```diff
@@ -0,0 +1,31 @@
+$spinkit-spinner-margin: 20px auto !default;
+$spinkit-spinner-color: #333 !default;
+
+.sk-wave {
+  $rectCount: 5;
```

Comment:
Name of variable `rectCount` should be written in all lowercase letters with hyphens instead of underscores